x/bank/handler_test.go

```diff
@@ -0,0 +1,18 @@
+package bank
+
+import (
+	"strings"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestInvalidMsg(t *testing.T) {
```

Comment:
wow, we didn't have any test for the handler ?

Reply:
Not directly, no. I believe most of the handler testing is done through simulation and app/abci module tests, but not directly.